INSTALL.md

```diff
@@ -20,22 +20,10 @@
 conda create --name maskrcnn_benchmark
 conda activate maskrcnn_benchmark
 
-# this installs the right pip and dependencies for the fresh python
-conda install ipython
```

Comment:
Please let's keep this `ipython`, it's always better to have it or else `pip` might point to the wrong `pip`